integrations/db/pgsql/src/test/java/io/helidon/integrations/db/pgsql/TestBaseDataSourceSubstitution.java

```diff
@@ -79,16 +80,16 @@ public String getDescription() {
         dataSrc.setProperty(PGProperty.APPLICATION_NAME, appNameProperty);
         BaseDataSourceSubstitution dataCopy = new BaseDataSourceSubstitution();
         dataCopy.initializeFrom(dataSrc);
-        assertEquals(databaseName, getPrivateField(dataCopy, "databaseName"));
-        assertEquals(user, getPrivateField(dataCopy, "user"));
-        assertEquals(password, getPrivateField(dataCopy, "password"));
+        assertThat(getPrivateField(dataCopy, "databaseName"), is(databaseName));
+        assertThat(getPrivateField(dataCopy, "user"), is(user));
+        assertThat(getPrivateField(dataCopy, "password"), is(password));
         int[] dataPortNumbers = (int[]) getPrivateField(dataCopy, "portNumbers");
         Properties dataProperties = (Properties) getPrivateField(dataCopy, "properties");
-        assertEquals(portNumbers.length, dataPortNumbers.length);
+        assertThat(dataPortNumbers.length, is(portNumbers.length));
```

Comment:
I can't use `arrayContaining` or similar because of 'int[]'. 
<img width="1669" alt="Screenshot 2022-11-27 at 10 46 54" src="https://user-images.githubusercontent.com/4740207/204126819-bc5f5b6a-d663-458a-a3d2-e3630397aeb3.png">